src/header/header.scss

```diff
@@ -190,181 +136,191 @@
         @include themed {
           background-color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-6));
         }
-
-        &:not(:focus-visible):not(:hover) {
-          @include themed {
-            background-color: t(iui-color-background-1);
-          }
-        }
       }
     }
-  }
 
-  .iui-header-button-icon {
-    margin: 0 $iui-l;
-    @include themed {
-      fill: t(iui-icons-color-actionable);
-    }
-  }
+    .iui-slim {
+      .iui-header-brand-icon {
+        width: $iui-icons-default;
+        height: $iui-icons-default;
+        margin: 0 $iui-l + $iui-xs;
+      }
 
-  > .iui-label {
-    margin-right: $iui-sm;
-    font-size: $iui-font-size-leading;
-    @media (prefers-reduced-motion: no-preference) {
-      transition: font-size $iui-speed-fast ease-out;
+      .iui-header-brand-label {
+        font-size: $iui-font-size;
+      }
     }
   }
 }
 
-@mixin iui-header-button-focus {
-  &:focus {
-    box-shadow: none;
-    outline-offset: -1px;
-
-    @include themed {
-      outline: 1px solid t(iui-color-foreground-primary);
-    }
-  }
-
-  &:focus:not(:focus-visible) {
-    outline: 0;
-  }
-}
-
-@mixin iui-header-button {
+@mixin iui-header-breadcrumbs-button {
+  @include iui-button;
+  @include iui-button-borderless;
   @include iui-button-size(large, borderless);
-
+  @include iui-header-button-reset;
   gap: 0;
   max-width: 25vw;
-  height: 100%;
-  border-radius: 0;
   overflow: hidden;
   padding-right: $iui-s;
 
-  @include iui-header-button-focus;
+  &-name,
+  &-description {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    transition: color $iui-speed-fast ease-out;
+  }
+
+  &-description {
+    height: $iui-baseline * 2;
+    @media (prefers-reduced-motion: no-preference) {
+      transition: height $iui-speed-fast ease-out;
+    }
+    @include themed {
+      opacity: t(iui-opacity-3);
+    }
+  }
 
   > .iui-button-label {
     text-align: left;
     font-size: $iui-font-size;
     overflow: hidden;
 
-    > div {
-      overflow: hidden;
-      text-overflow: ellipsis;
-    }
-
-    .iui-description {
-      height: $iui-baseline * 2;
-      @media (prefers-reduced-motion: no-preference) {
-        transition: all $iui-speed-fast ease;
-      }
-      @include themed {
-        opacity: t(iui-opacity-3);
-      }
+    @media (max-width: 768px) {
+      margin-left: 0;
     }
   }
 
   > .iui-button-icon {
     flex-shrink: 0;
+    box-sizing: content-box;
   }
 
   > * + * {
     margin-left: $iui-sm;
   }
 
   .iui-header-button-icon {
+    display: inline-flex;
     padding: $iui-xs;
+    box-sizing: content-box;
```

Comment:
What issue does this fix and how? Feels hacky and we shouldn't rely on it.

Reply:
In iTwinUI-Layouts Vilius is applying `* { box-sizing: border-box; }` & I know we plan on doing that across all `iui-` classes eventually.  In our layout, we are seeing this:
![Screen Shot 2022-02-18 at 4 31 43 PM](https://user-images.githubusercontent.com/849817/154764009-e1150c62-ea7a-4cba-a5ee-a6cc07aefa7d.png)
 
It should look like this & `content-box` fixes it:
![Screen Shot 2022-02-18 at 4 32 35 PM](https://user-images.githubusercontent.com/849817/154764113-1f78aec0-e920-468b-95cb-7c5dbfddbe54.png)